src/ethereum-providers/index.js

```diff
@@ -0,0 +1,61 @@
+import frame from './icons/Frame.png'
+import cipher from './icons/Cipher.png'
+import metamask from './icons/Metamask.png'
+import status from './icons/Status.png'
+
+// see the corresponding prop type, EthereumProviderType, in prop-types.js.
+const PROVIDERS = new Map(
```

Comment:
I believe it's worth adding Metamask as a mobile provider too.

Reply:
And, I have a concern: this cards are not actual selecting anything right? they're just to display different providers? may this confuse the users say if they're using Opera or Coinbase Wallet to believe the app won't work for them?